src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/ClusterAwareWriterFailoverHandler.java

```diff
@@ -278,7 +278,7 @@ public WriterFailoverResult call() {
       try {
         boolean success = false;
         while (!success) {
-          connectoToReader();
+          connectToReader();
```

Comment:
We already fixed this on introspection, I suggest we don't make this change again to main.